skimage/measure/mc_meta/visual_test.py

```diff
@@ -0,0 +1,72 @@
+""" 

```

Comment:
I'm not sure where this file should live.  How do we make sure it is runnable?  We probably need to update travis-CI, install visvis, render to an output buffer, and compare to a pre-existing image or hash of that image.